model/serviceProvider/LtiDeliveryServiceProvider.php

```diff
@@ -48,5 +50,14 @@ public function __invoke(ContainerConfigurator $configurator): void
                         ->args([[]])
                 ]
             );
+
+        $services
+            ->set(LtiConcurringSessionService::class, LtiConcurringSessionService::class)
```

Comment:
We can just inject parameters replacing the delivery execution service with `LtiDeliveryExecutionService::SERVICE_ID`

```suggestion
            ->set('MyCustomIdentifier', ConcurringSessionService::class)
```

Reply:
https://github.com/oat-sa/extension-tao-ltideliveryprovider/pull/387#discussion_r1565667043